manifests/charts/istio-control/istio-discovery/files/waypoint.yaml

```diff
@@ -173,14 +173,15 @@ spec:
           successThreshold: 1
           timeoutSeconds: 1
         securityContext:
-          privileged: true
+          privileged: false
           runAsGroup: 1337
           runAsUser: 0
           capabilities:
-            add:
-            - NET_ADMIN
-            - NET_RAW
+            drop:
+            - ALL
         volumeMounts:
+        - name: workload-socket
+          mountPath: /var/run/secrets/workload-spiffe-uds
```

Comment:
do we need to sign workload cert for waypoint?

Reply:
OK, maybe need to handle that before this pr.

Reply:
Any tips on where to look for this? Bit new to this, just got back

Reply:
It is expected to need this volume. Please keep it as is. Thanks!

Reply:
Yep I haven't touched anything yet, everything else I belive has been resolved from my end 🤔